lib/util/backend.dart

```diff
@@ -123,10 +123,12 @@ class Backend {
 
   /// Retrieves the current user role as a [String].
   static Future<String> getRole() async {
+    const String patientRoute = '/home-screen/home-screen';
     // these are thr routes we are going to forward the user to
-    Map<String, String> routes = <String, String>{
-      'Patient': '/home-screen/home-screen',
-      'Doctor': '/study-nurse-screen/menu-screen/menu-main-screen',
+    const Map<String, String> routes = <String, String>{
+      UserRoles.patient: patientRoute,
+      UserRoles.testPatient: patientRoute,
+      UserRoles.doctor: '/study-nurse-screen/menu-screen/menu-main-screen',
```

Comment:
I would also create a const variable for the path string.

Reply:
Resolved [98a284a]